app/views/users/edit.html.erb

```diff
@@ -1,5 +1,6 @@
 <% provide(:title, 'Edit user') %>
 <% provide(:button_text, 'Save changes') %>
+<p></p>
```

Comment:
@shaniajubelag Same comment in https://github.com/framgia/ph_sels_rails_shania/pull/8/files#diff-39ce76d94a7eb2d6282c0ff8a4ce44a9R2